src/main/java/com/exactpro/th2/FixHandler.java

```diff
@@ -861,7 +861,7 @@ public int getBodyLength(ByteBuf message) {
     public String getTime() {
         DateTimeFormatter formatter = settings.getSendingDateTimeFormat();
         if (formatter == null) {
-            formatter = DateTimeFormatter.ofPattern(FIX_DATE_TIME_FORMAT_MS);
+            formatter = DEFAULT_FORMATTER;
         }
```

Comment:
We can make `DEFAULT_FORMATTER` a default value for `sendingDateTimeFormat` option to avoid this branch at all